src/errors.rs

```diff
@@ -88,7 +89,14 @@ pub enum RustupError {
     ToolchainNotInstallable(String),
     #[error("toolchain '{0}' is not installed")]
     ToolchainNotInstalled(String),
-    #[error("no override and no default toolchain set")]
+    #[error("{} have not specified a toolchain version to use.\n{}\n{}",
+        match utils::current_name() {
+            Some(arg0) if arg0.starts_with("rustup") => "you".to_string(),
+            None => "you".to_string(),
+            Some(arg0) => format!("you are trying to run {} via rustup, but", arg0)
+        },
+        "help: run 'rustup default stable' to download the latest stable release of Rust and set it as your default toolchain.",
+        "      You can also configure toolchains per-project: https://rust-lang.github.io/rustup/overrides.html")]
```

Comment:
I think for this situation, which we expect when people get rustup from somewhere else like distro tooling, they may often be using existing rust projects and won't be in a position to get an override specified. Lets not point them at the override: setting a default is the most generic and useful solution, and people can always read the docs to know more.

Reply:
Maybe remove the text, but keep the link? I want the error to have some kind of answer to "what should I do if I want something else?".

Reply:
I agree they should get an answer. I'm proposing that we don't get into the larger set of different possible answers: env variables, toolchain files, (in future) rust-version in Cargo.toml. And instead, we just say 'set a default'.